packages/tasks/src/tasks/audio-classification/inference.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Generated on 2024-01-19T14:59:10.562Z
+ */
+
+/**
+ * Inputs for Audio Classification inference
+ */
+export interface AudioClassificationInput {
```

Comment:
Audio data is usually pass through `data`

https://huggingface.co/docs/api-inference/detailed_parameters#audio-classification-task

Reply:
Same for images. The jsonschema cannot specify this since sending as raw data and sending as json are 2 different things. So for now it's kind of a blind spot. If we provide an openapi schema for our APIs in the future, then it will be possible to document it. Openapi easily integrates with jsonschema so having them is already a first good step.

(difference between a jsonschema as in this PR and an openapi description is that this PR describes objects with their attributes while the openapi description with include stuff like server routes, accepted headers, etc.)

Reply:
Yes - sorry for the delay in answering

Leaving the image/audio data as `unknown` was intentional, to give more flexibility to the libraries.
Image & audio data can be passed in several different forms (raw binary data, path to a local or remote file, base64 encoded data...) and I did not want to constrain downstream users of those types into one single representation.

> (difference between a jsonschema as in this PR and an openapi description is that this PR describes objects with their attributes while the openapi description with include stuff like server routes, accepted headers, etc.)

Yes that is correct, there will be some additional work necessary to generate an OpenAPI spec for an inference API (including actually specifying how we expect the binary data to be represented)